From b3f2636f44eea1a8b6fbf892d2daa611cff9d4af Mon Sep 17 00:00:00 2001
From: Alyssa Ross <hi@alyssa.is>
Date: Wed, 1 Apr 2020 14:30:51 +0000
Subject: [PATCH] Fix missing header without LZO

lib/common/compress.c: In function 'compressor_get_default':
lib/common/compress.c:39:2: warning: implicit declaration of function 'assert' [-Wimplicit-function-declaration]
   39 |  assert(0);
      |  ^~~~~~
lib/common/compress.c:8:1: note: 'assert' is defined in header '<assert.h>'; did you forget to '#include <assert.h>'?
    7 | #include "common.h"
  +++ |+#include <assert.h>
    8 |
---
 lib/common/compress.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/common/compress.c b/lib/common/compress.c
index ea7b71f..f16f080 100644
--- a/lib/common/compress.c
+++ b/lib/common/compress.c
@@ -5,6 +5,7 @@
  * Copyright (C) 2019 David Oberhollenzer <goliath@infraroot.at>
  */
 #include "common.h"
+#include <assert.h>
 
 static int cmp_ids[] = {
 	SQFS_COMP_XZ,
-- 
2.24.1

